c/stream_tests.c

```diff
@@ -0,0 +1,253 @@
+#define _POSIX_C_SOURCE 200809L /* fdopen(), strsignal() need this */
```

Comment:
Well, this could be reformed as a test. It was just easier to write without learning cunit at the same time. But I agree, it would be very nice to drive this from python.

Reply:
Removed this for now.